content/en/blog/2020/mosn-proxy/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Using MOSN with Istio: an alternative data plane"
+subtitle: "A Cloud Native Proxy for Edge or Service Mesh"
+description: "An alternative sidecar proxy for Istio."
+publishdate: 2020-07-25
+attribution: "Wang Fakang (mosn.io)"
+keywords: [mosn,sidecar,proxy]
+---
+
+[MOSN](https://github.com/mosn/mosn) (Modular Open Smart Network) is a network proxy server written in GoLang. It was built at [Ant Group](https://www.antfin.com) as a sidecar/API Gateway/cloud-native Ingress/Layer 4 or Layer 7 load balancer etc. Over time, we've added extra features, like a multi-protocol framework, multi-process plug-in mechanism, a DSL, and support for the [xDS APIs](https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol). Supporting xDS means we are now able to use MOSN as the network proxy for Istio.
+
+## Background
+
+In the service mesh world, using Istio as the control plane has become the mainstream. Because Istio was built on Envoy, it uses Envoy's data plane [APIs](https://blog.envoyproxy.io/the-universal-data-plane-api-d15cec7a) (collectively known as the xDS APIs). These APIs have been standardized separately from Envoy, and so by implementing them in MOSN, we are able to drop in MOSN as a replacement for Envoy. Istio's integration of third-party data planes can be implemented in three steps, as follows.
+
+- Implement xDS protocols to fulfill the capabilities for data plane related services.
+- Build `proxyv2` images using Istio's script and set the relevant `SIDECAR` and other parameters.
+- Specify a specific data plane via the `istioctl` tool and set the proxy-related configuration.
+
```

Comment:
Please add a "## Support" section after "## Background" saying that MOSN and its integration with Istio is not a configuration supported by the Istio project.  Also, please describe where users who deploy Istio+MOSN should send questions and support requests (these should be links to MOSN resources, not Istio resources)

Reply:
> Please add a "## Support" section after "## Background" saying that MOSN and its integration with Istio is not a configuration supported by the Istio project. Also, please describe where users who deploy Istio+MOSN should send questions and support requests (these should be links to MOSN resources, not Istio resources)

Thanks your review, the support links have been written in both `Moving forward` and `Learn More` sections.

Reply:
I've added a suggestion above in the intro section.  Is that good, Josh?

Reply:
@duderino @craigbox I have added a separate section about `Support` to make it clearer. Thanks your suggestion.

Reply:
from Slack: "I think TOC would prefer you explicitly say that the Istio project doesn’t support this config"

Reply:
Ok, Done.

Reply:
I'm happy with this statement ```This configuration is not supported by the Istio project; for help, please see [Learn More](#learn-more) below.``` thanks